### ISSUE TITLE
Running `./traceroute` along would cause panic

### ISSUE THREAD

**Issue by 10000TB** [u-root/u-root#3220]
**Describe the bug**

```
xuehaohu@worldpeace:~/go/src/github.com/u-root/u-root/cmds/exp/traceroute$ ./traceroute
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.parseFlags({0xc0000161b0, 0x1, 0x1})
	/usr/local/google/home/xuehaohu/go/src/github.com/u-root/u-root/cmds/exp/traceroute/main.go:54 +0x64f
main.run({0xc0000161b0?, 0xc00007e060?, 0x6337d8?})
	/usr/local/google/home/xuehaohu/go/src/github.com/u-root/u-root/cmds/exp/traceroute/main.go:80 +0x18
main.main()
	/usr/local/google/home/xuehaohu/go/src/github.com/u-root/u-root/cmds/exp/traceroute/main.go:91 +0x28
```

**To Reproduce**

Use HEAD commit as follows, build `traceroute`, and run just `traceroute` w/o flags.

```
xuehaohu@worldpeace:~/go/src/github.com/u-root/u-root/cmds/exp/traceroute$ git log -1
commit e3b3cae21292a1c40d75d9066b1aeec446f790ea (HEAD -> main, origin/main, origin/HEAD)
Author: Ronald G Minnich <rminnich@gmail.com>
Date:   Sat Nov 30 19:55:00 2024 -0800

    Fix race in gosh_test.go

    Signed-off-by: Ronald G Minnich <rminnich@gmail.com>
xuehaohu@worldpeace:~/go/src/github.com/u-root/u-root/cmds/exp/traceroute$ go build .
xuehaohu@worldpeace:~/go/src/github.com/u-root/u-root/cmds/exp/traceroute$ ./traceroute
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.parseFlags({0xc0000161b0, 0x1, 0x1})
	/usr/local/google/home/xuehaohu/go/src/github.com/u-root/u-root/cmds/exp/traceroute/main.go:54 +0x64f
main.run({0xc0000161b0?, 0xc00007e060?, 0x6337d8?})
	/usr/local/google/home/xuehaohu/go/src/github.com/u-root/u-root/cmds/exp/traceroute/main.go:80 +0x18
main.main()
	/usr/local/google/home/xuehaohu/go/src/github.com/u-root/u-root/cmds/exp/traceroute/main.go:91 +0x28
xuehaohu@worldpeace:~/go/src/github.com/u-root/u-root/cmds/exp/traceroute$
```

**Expected behavior**

no panic.

**Additional context**

**Comment by ChriMarMe** [comment]
https://github.com/u-root/u-root/pull/3221

**Comment by jensdrenhaus** [comment]
Fixed in #3221
